test/tools/router_check/test/route_tests.sh

```diff
@@ -20,14 +20,14 @@ done
 # --useproto needs the test schema as a validation.proto message.
 for t in "${TESTS[@]}"
 do
-  TEST_OUTPUT=$("${PATH_BIN}" "${PATH_CONFIG}/${t}.yaml" "${PATH_CONFIG}/${t}.golden.proto.json" "--details" "--useproto")
+  TEST_OUTPUT=$("${PATH_BIN}" "-c" "${PATH_CONFIG}/${t}.yaml" "-t" "${PATH_CONFIG}/${t}.golden.proto.json" "--details" "--useproto")
```

Comment:
Is this a breaking change for all the old tool? Does it still work w/o the explicit flags? If so, can we test that?

Reply:
@mattklein123 The old tool still works as demonstrated in the example here. https://github.com/envoyproxy/envoy/blob/8e2a1f49d869af658bc7ddda327c55b176ec8752/test/tools/router_check/test/route_tests.sh#L23

The breaking change is only for usages of the new --useproto code path. Since that was committed last Wednesday, I hope there is not a lot of usage yet. If you suspect there might be usage of --useproto in the community already, we could make the code handle the situation with a few if-else cases. Let me know WDYT.

Reply:
OK I see, sorry, that's fine.